src/library/dao/cue.cpp

```diff
@@ -7,6 +7,10 @@
 #include "library/dao/cue.h"
 #include "util/assert.h"
 
+namespace {
+    const QColor defaultColor = QColor("#FF0000");
```

Comment:
Looks good ;) But according to Mixxx's coding style the constant should be named "kDefaultColor"